<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWalletDialog</class>
 <widget class="QDialog" name="CreateWalletDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a new wallet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_pages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_wallet">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Wallet name :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_name"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_key">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Private key</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="edit_secretkey_path"/>
            </item>
            <item>
             <widget class="QPushButton" name="button_import">
              <property name="text">
               <string>Import a secret key</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>OR</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_fingerprint">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_generate">
              <property name="text">
               <string>Generate a secret key</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_node">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Main node :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edit_address"/>
         </item>
         <item>
          <widget class="QSpinBox" name="spinbox_port">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>8881</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_previous">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_import</sender>
   <signal>clicked()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>open_import_key()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_generate</sender>
   <signal>clicked()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>open_generate_key()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_address</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinbox_port</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_previous</sender>
   <signal>clicked()</signal>
   <receiver>CreateWalletDialog</receiver>
   <slot>previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_import_key()</slot>
  <slot>open_generate_key()</slot>
  <slot>check()</slot>
  <slot>next()</slot>
  <slot>previous()</slot>
 </slots>
</ui>
